db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'rubygems'
+require 'nokogiri'
+require 'open-uri'
+
+page = Nokogiri::HTML(open("http://www.en365.ru/top1000b.htm"))
+rows = page.css('td#middle tr')
+
+rows.each do |row|
+  cols = row.css('td').map{|td| td.text}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Pass `&:text` as an argument to `map` instead of a block.